app/controllers/concerns/posts_controller.rb

```diff
@@ -1,16 +1,20 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource except:
+  %i[index show]
+
   def index
     @posts = Post.includes(:author).where(author_id: params[:user_id])
     @user = User.find(params[:user_id])
-    @posts = @posts.paginate(page: params[:page], per_page: 3)
   end
 
   def show
-    @post = Post.includes(:author).find_by(author_id: params[:user_id], id: params[:id])
+    # @post = Post.includes(:author).find_by(author_id: params[:user_id], id: params[:id])
```

Comment:
- [ ] Same here as pointed out in the previous comment :point_up_2: Kindly remove all the commented-out code from your code base.